tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetCore5AsmTelemetry.cs

```diff
@@ -0,0 +1,105 @@
+// <copyright file="AspNetCore5AsmTelemetry.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+#pragma warning disable SA1402 // File may only contain a single class
+#pragma warning disable SA1649 // File name must match first type name
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SimpleEmail.Model;
+using Datadog.Trace.AppSec.Rcm.Models.AsmFeatures;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.ManualInstrumentation.Configuration.TracerSettings;
+using Datadog.Trace.ClrProfiler.IntegrationTests;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests
+{
+    public class AspNetCore5AsmTelemetry : AspNetBase, IClassFixture<AspNetCoreTestFixture>
+    {
+        public AspNetCore5AsmTelemetry(ITestOutputHelper outputHelper)
+            : base("AspNetCore5", outputHelper, "/shutdown", testName: nameof(AspNetCore5AsmTelemetry))
+        {
+            // telemetry metric events under test are sent only when using managed trace exporter
+            SetEnvironmentVariable(ConfigurationKeys.TraceDataPipelineEnabled, "false");
```

Comment:
That's not true 😄 You're reading configuration values, and those are sent regardless. It's only "trace metrics" e.g. "number of traces sent" etc which aren't emitted with the data pipeline
```suggestion
```

Reply:
This is copy pasta from `TelemetryTests`